src/components/MaterialUI/muiThemes.js

```diff
@@ -676,8 +676,8 @@ const setThemeOverrides = theme => ({
 			border: "none",
 		},
 	},
-	MuiExpansionPanelSummary: {
-		...theme.MuiExpansionPanelSummary,
+	MuiAccordionSummary: {
```

Comment:
Would it hurt to keep both ? We never know...

Reply:
Doesn't make sense. Those blobal classes not exists for Accordion. And they have plans to remove Expansion Panel soon